packages/expo-task-manager/android/src/main/java/expo/modules/taskManager/TaskJobService.java

```diff
@@ -0,0 +1,21 @@
+package expo.modules.taskManager;
+
+import android.annotation.SuppressLint;
+import android.app.job.JobParameters;
+import android.app.job.JobService;
+import android.content.Context;
+
+@SuppressLint("NewApi")
```

Comment:
```suggestion
```

I guess it's from pre-API21 times? When removed, AS doesn't complain about anything in that file